src/components/layout/LayoutHeader.tsx

```diff
@@ -163,10 +167,19 @@ export const LayoutHeader = () => {
           />
 
           <MainNav />
+          <ConnectWallet />
 
           <div className="LabLayout__header__settings">
             <Hydration>
-              <ThemeSwitch storageKeyId="stellarTheme:Laboratory" />
+              <ThemeSwitch
+                storageKeyId={LOCAL_STORAGE_SAVED_THEME}
+                onActionEnd={(isDarkMode) => {
+                  const theme = isDarkMode
+                    ? "sds-theme-dark"
+                    : "sds-theme-light";
+                  setTheme(theme);
+                }}
+              />
```

Comment:
We should probably create a helper that renders this in both places. They have more props now, but they are the same.